x-pack/plugin/esql/qa/testFixtures/src/main/resources/stats.csv-spec

```diff
@@ -1552,7 +1552,7 @@ s2point1:d | s_mv:i | languages:i
 2.1        | 3      | null
 ;
 
-evalOverridingKey
+evalOverridingKey#[skip:-8.13.1,reason:fixed in 8.13.2]
```

Comment:
Added in https://github.com/elastic/elasticsearch/pull/107131, which was backported to 8.13.2 but not earlier.